bot/exts/evergreen/avatar_modification/avatar_modify.py

```diff
@@ -121,6 +121,42 @@ async def eightbit_command(self, ctx: commands.Context) -> None:
 
         await ctx.send(embed=embed, file=file)
 
+    @avatar_modify.command(name="reverse", root_aliases=("reverse",))
+    async def reverse(self, ctx: commands.Context, *, text: t.Optional[str]) -> None:
+        """
+        Reverses the sent text.
+
+        If no text is provided, the user's profile picture will be reversed.
+        """
+        if not text:
```

Comment:
The changes look good now :) One small comment I have is that we can avoid this huge if statement here by doing the unhappy path first IE
```py
if text:
    await ctx.send(f"> {text[::-1]}", allowed_mentions=discord.AllowedMentions.none())
    return
``` and then doing the rest of the code. This will avoid the large nested block

Reply:
alright, gimme a few seconds